src/index.js

```diff
@@ -1,14 +1,18 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
```

Comment:
- [ ] I get the error below when I try to run your app and this is because you are importing an ```index.css``` file here but there is no such file in your codebase. Kindly delete this file

![Screenshot 2023-01-27 at 08 24 59 (2)](https://user-images.githubusercontent.com/64774936/215031262-d5852dac-1771-4cd7-a512-69fac44927ff.png)

Reply:
Hi @Ghiftee 
The issue is resolved now, thank you.